src/server/clients_info.rs

```diff
@@ -75,10 +75,10 @@ impl ClientsInfo {
     }
 }
 
-pub(super) struct ClientInfo {
+pub struct ClientInfo {
```

Comment:
Same.

Reply:
Sorry, missed it.
But I'm not sure if exporting `remove_despawned` to users is a good idea...